tests/DependencyInjection/SonataAdminExtensionTest.php

```diff
@@ -42,6 +42,27 @@ protected function setUp(): void
         ;
     }
 
+    public function testHasCoreServicesAlias(): void
+    {
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Admin\Pool');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Route\AdminPoolLoader');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Admin\AdminHelper');
+        $this->assertContainerBuilderHasService('Sonata\AdminBundle\Filter\FilterFactory');
```

Comment:
This class implements `FilterFactoryInterface`. Please, define a service alias for that interface pointing to the default implementation; and do the same for the other classes here that are implementing an interface.

Reply:
Done